packages/playwright-test/src/index.ts

```diff
@@ -25,6 +25,7 @@ import { GridClient } from 'playwright-core/lib/grid/gridClient';
 export { expect } from './expect';
 export const _baseTest: TestType<{}, {}> = rootTestType.test;
 import * as outOfProcess from 'playwright-core/lib/outofprocess';
+export { webServer } from './webServerPlugin';
```

Comment:
What do we want the public export to be? It's currently:

```ts
import { webServer } from '@playwright/test';
```

I don't mind this now, but if we add more official plugins, I think we'd want a sub-package:

```ts
import { webServer } from '@playwright/test/plugins';
```